src/pages/pickups.tsx

```diff
@@ -101,19 +119,39 @@ function AcceptJobButton({trackable, user}:{trackable:Trackable, user:User}) {
     }
 
     return (
-        <Button onClick={onClick}>Accept Job</Button>
+        <Button isLoading={loading} onClick={onClick}>Accept Job</Button>
+    )
+}
+
+function PickupAddr({addr}:{addr:{street:string,cityStateZip:string}|undefined}) {
+    if (!addr) {
+        return <Text>Unknown pickup address</Text>
+    }
+    return (
+        <Text> 
+            {addr.street}<br/>
+            {addr.cityStateZip}
+        </Text>
     )
 }
 
 function TrackableCollection({ trackables,user }: { trackables: Trackable[],user:User }) {
 
+
     const trackableElements = trackables.map((trackable: Trackable) => {
+        const firstUpdate = (trackable.updates!.edges!)[0]
```

Comment:
Is creation order preserved on these? [I sorted them by date on another page](https://github.com/quorumcontrol/simple-tracker/blob/1188fcf099ed9b2d8a8bd4e2f90b69ddb39989ea/src/pages/donationStatus.tsx#L59) just in case, but maybe it wasn't necessary.